src/dsp_tools/commands/excel2xml/excel2xml_lib.py

```diff
@@ -1883,41 +1811,286 @@ def make_video_segment(  # noqa: D417 (undocumented-param)
+    """
+    Make a `<relatesTo>` property for a `<video-segment>` or `<audio-segment>`.
+
+    Args:
+        target_id: ID of the related resource
+        permissions: defaults to "prop-default".
+        comment: optional comment for this property. Defaults to None.
+        calling_resource: the name of the parent resource (for better error messages)
+
+    Warns:
+        - If the target_id is not a valid string
 
     Examples:
-        >>> excel2xml.create_interval_value("0:01:00", "0:02:00")
-        "60:120"
-        >>> excel2xml.create_interval_value("0:01:00.5", "0:01:00.6")
-        "60.5:60.6"
-    """
-    start_match = regex.search(r"^(\d+):([0-5][0-9]):([0-5][0-9](?:\.[0-9]+)?)$", start)
-    end_match = regex.match(r"^(\d+):([0-5][0-9]):([0-5][0-9](?:\.[0-9]+)?)$", end)
-    if not start_match or not end_match:
-        raise ValueError("The start and end values must be in the format 'H(H):MM:SS(.s)'")
-    start_h, start_m, start_s = start_match.groups()
-    end_h, end_m, end_s = end_match.groups()
-    start_seconds = int(start_h) * 3600 + int(start_m) * 60 + float(start_s)
-    end_seconds = int(end_h) * 3600 + int(end_m) * 60 + float(end_s)
-    start_seconds = int(start_seconds) if start_seconds.is_integer() else start_seconds
-    end_seconds = int(end_seconds) if end_seconds.is_integer() else end_seconds
-    if start_seconds > end_seconds:
-        raise ValueError("The start value must be smaller than the end value")
-    return f"{start_seconds}:{end_seconds}"
+        >>> video_segment = excel2xml.make_video_segment("label", "id")
+        >>> video_segment.append(excel2xml.make_relatesTo_prop("resource_id"))
+        >>> video_segment.append(excel2xml.make_relatesTo_prop("other_resource_id)
+        >>> # add other properties
+        >>> root.append(video_segment)
+
+    Returns:
+        an etree._Element that can be appended to an audio/video segment with `segment.append(make_relatesTo_prop(...))`
+    """
+    if not isinstance(target_id, str) or not check_notna(target_id):
+        msg = (
+            f"Validation Error in resource '{calling_resource}', property 'relatesTo': "
+            f"The following doesn't seem to be a valid ID of a target resource: '{target_id}'"
+        )
+        warnings.warn(DspToolsUserWarning(msg))
+    prop = etree.Element("{%s}relatesTo" % xml_namespace_map[None], permissions=permissions)
+    if comment:
+        prop.set("comment", comment)
+    prop.text = target_id
+    return prop
+
+
+def make_hasSegmentBounds_prop(
+    start: int | float,
+    end: int | float,
+    permissions: str = "prop-default",
+    comment: str | None = None,
+    calling_resource: str = "",
+) -> etree._Element:
+    """
+    Make a `<hasSegmentBounds>` property for a `<video-segment>` or `<audio-segment>`.
+
+    Args:
+        start: start, in seconds, counted from the beginning of the audio/video
+        end: end, in seconds, counted from the beginning of the audio/video
```

Comment:
i would add `segment` at the end, above too, because this way it sounds like you need to enter the entire video length.

Reply:
To be consistent, I also changed it in the attribute of the XML. What do you think? Here how it looks like: https://github.com/dasch-swiss/dsp-tools/commit/286d404b1e65249a9821d7c47568887c0a0b13d4